lib/taskana-core/src/test/java/acceptance/jobs/TaskUpdatePriorityJobAccTest.java

```diff
@@ -31,10 +34,14 @@ void before() throws Exception {
     // TODO split test class into readOnly & modifying tests to improve performance
     resetDb(true);
 
-    taskanaEngineConfiguration.setPriorityJobActive(true);
-    taskanaEngineConfiguration.setPriorityJobBatchSize(20);
-    taskanaEngineConfiguration.setPriorityJobRunEvery(Duration.ofMinutes(30));
-    taskanaEngineConfiguration.setPriorityJobFirstRun(Instant.parse("2007-12-03T10:15:30.00Z"));
+    taskanaEngineConfiguration =
```

Comment:
We should rename all variables aswell. -> https://taskana.atlassian.net/browse/TSK-2002